changelogs/fix-6471

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
@moon0326 I'm not very familiar with the new changelogger.  Could you take a look to see if this looks okay?

Reply:
Change log format is good here 👍